utils/filesystem.go

```diff
@@ -90,14 +90,15 @@ func DeleteTempFile(tempFilePath string) (boolean bool, err error) {
 func PingHealth(healthEndpoint string) bool {
 	var started = false
 	fmt.Println("Waiting for Codewind to start")
+	port := GetPFEPort()
 	for i := 0; i < 120; i++ {
-		resp, err := http.Get(healthEndpoint)
+		resp, err := http.Get("http://localhost:" + port + healthEndpoint)
 		if err != nil {
 			fmt.Printf(".")
 		} else {
 			if resp.StatusCode == 200 {
 				fmt.Println("\nHTTP Response Status:", resp.StatusCode, http.StatusText(resp.StatusCode))
-				fmt.Println("Codewind successfully started on port " + GetPort())
+				fmt.Println("Codewind successfully started on port " + port)
```

Comment:
Again, should this be the full url we did the http.Get() on?

Reply:
Not sure we should be broadcasting what we use as a health endpoint - perhaps a security risk? I've continued to mirror what the status message returns.

Reply:
Not a security risk, it's running on their box after all but you're right showing the same as the status message is better.